### ISSUE TITLE
Cannot install `0.2.2` of core and `0.2.3` of community due to peer resolution error

### ISSUE THREAD

**Issue by jsumners-nr** [langchain-ai/langchainjs#5572]
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

With Node 18 or Node 20, try to:

```sh
$ npm install --no-save --no-package-lock --no-audit --no-fund @langchain/core@0.2.2 @langchain/community@0.2.3
```

### Error Message and Stack Trace (if applicable)

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: undefined@undefined
npm ERR! Found: @google-ai/generativelanguage@0.2.1
npm ERR! node_modules/@google-ai/generativelanguage
npm ERR!   peerOptional @google-ai/generativelanguage@"^0.2.1" from langchain@0.1.37
npm ERR!   node_modules/langchain
npm ERR!     peerOptional langchain@"~0.1.19" from @getzep/zep-cloud@1.0.3
npm ERR!     node_modules/@getzep/zep-cloud
npm ERR!       @getzep/zep-cloud@"^1.0.3" from the root project
npm ERR!       1 more (@langchain/community)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional @google-ai/generativelanguage@"^2.5.0" from @langchain/community@0.2.3
npm ERR! node_modules/@langchain/community
npm ERR!   @langchain/community@"0.2.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Description

The peer dependency tree cannot be resolved satisfactorily when trying to install `@langchain/core@0.2.2` and `@langchain/community@0.2.3`. The issue is that:

1. `@langchain/community@0.2.3` has a peer dependency on `@google-ai/generativelanguage@2.5.0`
2. `@langchain/community@0.2.3` has a peer dependency on `@getzep/zep-cloud@1.0.3`
3. `@getzep/zep-cloud@1.0.3` has a peer dependency on `langchain@0.1.19`
4. `langchain@0.1.37` has a peer dependency on `@google-ai/generativelanguage@0.2.1`

In other words, `npm` cannot figure out how to satisfy the `@google-ai/generativelanguage` dependency with the provided graph.

### System Info

```
❯ node --version
v20.13.1
```

and

```
❯ node --version
v18.20.3
```

**Comment by bracesproul** [comment]
I just cut a release candidate for `@langchain/community` [v0.2.4-rc.1](https://www.npmjs.com/package/@langchain/community/v/0.2.4-rc.1)

I tested locally and it fixed the issue for me, could you confirm on your end? If so I'll upgrade it to an actual release.

```bash
npm install --no-save --no-package-lock --no-audit --no-fund @langchain/core@0.2.2 @langchain/community@0.2.4-rc.1
```

**Comment by jacoblee93** [comment]
@bracesproul are we even using `@google-ai/generativelanguage` in `langchain` anymore? I think it might just be in community?

Could we just clear the dep?

**Comment by bracesproul** [comment]
@jsumners-nr `@langchain/community@0.2.4` published

**Comment by hangyeoldora** [comment]
@bracesproul ```npm i @langchain/community@0.2.4``` it doesn't work... <br/>I installed it using this command ```npm install @langchain/core@0.2.2 @langchain/community@0.2.3 --legacy-peer-deps``` written above.

**Comment by bracesproul** [comment]
@hangyeoldora `@langchain/community@0.2.3` is the old version which has a known bug that's fixed in `@langchain/community@0.2.4`. If you run this command to install do you still experience errors?
```
npm install --no-save --no-package-lock --no-audit --no-fund @langchain/core@0.2.2 @langchain/community@0.2.4
```

**Comment by hangyeoldora** [comment]
@bracesproul  It works well after deleting the 0.2.3 version and installing it. I have successfully installed it now. thanks
